focus_areas.md

```diff
@@ -0,0 +1,51 @@
+# Focus Areas
+
+Concise definitions of what is in scope for each focus area:
+
+*  interop-2022-cascade:
+    * Cascade Layers, CSS Cascade Level 5
+*  interop-2022-color:
+    * CSS `<color>` type, CSS Color Level 4
+*  interop-2022-contain:
+    * CSS contain property, CSS Containment Level 1, excluding the `style` keyword value
+*  interop-2022-dialog:
+    * the dialog element and its open attribute,
+    * the IDL at https://html.spec.whatwg.org/review-drafts/2021-07/#the-dialog-element,
+    *  any integration into other web platform features (e.g., CSS Animations)
+*  interop-2022-forms:
+    * CSS `appearance` property, CSS UI 4
+    * [FIXME: box sizing of form controls, or something to cover "intrinsic margins" in WebKit previously?]
+    * DOM click events on non-disabled form controls
+    * Form submission
+    * Form validation
+    * Text field seletion
+    * [FIXME: Some subset of rendering?]
+*  interop-2022-scrolling:
+    * CSS Overscroll Behavior Module Level 1;
+    * CSS Scroll Snap Module Level 1;
+    * dictionary ScrollOptions and its usage, CSSOM View Level 1
+*  interop-2022-subgrid:
+    * `subgrid` value for grid-template-columns and grid-template-rows, CSS Grid Level 2
+*  interop-2022-text:
+    * iso-2022-jp/gbk/gb18030 encoding/decoding
+    * font-variant (including longhands), CSS Fonts Level 4 [FIXME: or should this be level 3?]
+    * ic unit, CSS Values and Units Level 4
+*  interop-2022-viewport:
+    * Viewport-percentage Lengths, CSS Values and Units Level 4
+*  interop-2022-webcompat:
+    * text-decoration with subelements
+    * text-decoration-offset: 0px
+    * /css/CSS2/floats/floats-wrap-bfc-with-margin-001.html (but not any of the further floats-wrap-bfc-with-margin-*)
```

Comment:
I don't really know how to define this part of webcompat without referring to the specific test